sycl/test-e2e/InvokeSimd/Feature/invoke_simd_struct.cpp

```diff
@@ -2,34 +2,23 @@
 // REQUIRES: gpu && linux
 // UNSUPPORTED: cuda || hip
 //
+// TODO: enable after Jira issue resolved
+// XFAIL: gpu
+//
```

Comment:
How does it fail? Is it compilation failure or runtime one? We've had some issues with CI stability that we suspect might be due to XFAIL'ed tests hitting bugs outside SYCL RT, so, in general, we prefer to use UNSUPPORTED instead of XFAIL.

Reply:
Test fails on runtime. Failure does not depend on the presence of XFAIL label.